src/styles/card.scss

```diff
@@ -0,0 +1,106 @@
+@use './variables' as *;
+
+.product {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 0 16px;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), properties within a selector should be ordered alphabetically. Please rearrange the properties of the '.product' selector accordingly.